src/editor/FileLoader.java

```diff
@@ -0,0 +1,65 @@
+package editor;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+
+public class FileLoader {
+
+    String loadFile(String fileName)  {
+        File file = new File(fileName);
+
+        /*
+            If file exists and not empty, read it
+         */
+        if(file.isFile()) {
+            try {
+               return Files.readString(Paths.get(fileName));
+            } catch (IOException e) {
+                e.printStackTrace();
+                return "File is Empty!";
+            }
+        } else {
+            return "File Not Found!";
+        }
+    }
+
+    void saveFile(String fileName, String text) {
+
+        Scanner scName = new Scanner(fileName);
+        /*
+            If file name is empty
+            use default one
+        */
+        if( !scName.hasNext()) {
```

Comment:
You did a good job by decomposing all work with files into a separate class!  But try to be consistent with your code style - for example, you typed somewhere a spacebar after the opening brace and somewhere not typed.